hack/auth-config-template.yaml

```diff
@@ -0,0 +1,12 @@
+# see Region Identifier in https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm
+# for a list of regions
+region: <region>
+tenancy: <tenancy OCID>
+user: <user OCID>
+key: |
+  -----BEGIN RSA PRIVATE KEY-----
+  <key info> 
+  -----END RSA PRIVATE KEY-----
+fingerprint: <fingerprint>
+
+useInstancePrincipals: false
```

Comment:
Aside: do we support the use of instance principals? If so, wouldn't this be true in that case?

Reply:
I believe they are supported, but not running locally in a kind cluster.

Reply:
What if the kind cluster was running on an Oracle Linux or Ubuntu instance in OCI?